packages/svelte/src/internal/client/dom/blocks/each.js

```diff
@@ -601,8 +596,6 @@ function create_item(anchor, prev, value, key, index, render_fn, flags, get_coll
 
 		item.e = branch(() => render_fn(/** @type {Node} */ (anchor), v, i, get_collection));
 
-		item.e.prev = prev && prev.e;
-
 		if (prev !== null) {
```

Comment:
Maybe add a comment here elaborating why we set next but not prev - feels kinda natural at first to set both directions